code/processes/wdb.q

```diff
@@ -28,11 +28,15 @@ writedownmode:@[value;`writedownmode;`default];                            /-the
                                                                            /-                                      at EOD the data will be sorted and given attributes according to sort.csv before being moved to hdb
                                                                            /- 2. partbyattr                -       the data is partitioned by [ partitiontype ] and the column(s) assigned the parted attributed in sort.csv
                                                                            /-                                      at EOD the data will be merged from each partiton before being moved to hdb
+                                                                           /- 3. partbyenum                -       the data is partitioned by [ partitiontype ] and a symbol column with parted attribution assigned in sort.csv
+                                                                           /-                                      at EOD the data will be merged from each partiton before being moved to hdb
+partwritemodes:`partbyattr`partbyenum;
```

Comment:
this section at the top of scripts in TorQ is conventionally reserved for config - I don't think this belongs here